ArcGISRuntimeSDKQt_QMLSamples/Layers/OSM_Layer/README.metadata.json

```diff
@@ -13,7 +13,6 @@
         "map",
         "open",
         "street",
-        "BasemapOpenStreetMap",
```

Comment:
You will need to add `Basemap` in the relevant api as well as the keywords for the metadata since you added it to the readme.

Reply:
@tanneryould  I'll change this one myself.